conan/tools/gnu/autotoolstoolchain.py

```diff
@@ -91,6 +92,8 @@ def __init__(self, conanfile, namespace=None, prefix="/"):
                 self.apple_arch_flag = "-arch {}".format(apple_arch) if apple_arch else None
                 # -isysroot makes all includes for your library relative to the build directory
                 self.apple_isysroot_flag = "-isysroot {}".format(sdk_path) if sdk_path else None
+            self.cross_build = {"host": {"arch": arch_host, "triplet": self._host},
+                                "build": {"arch": arch_build, "triplet": self._build}}
```

Comment:
Why not defining and giving public access directly to ``self._host`` and ``self._build``?